frontend/src/api-calls.ts

```diff
@@ -1,20 +1,20 @@
 import axios from "axios";
 import {Spot} from "./models/Spot";
-import {UserAuth} from "./models/User";
+import {NewUserRequest, UserLoginRequest} from "./models/User";
 
-export function getAccessToken(){
+export function getAccessToken() {
     return axios.get("/api/mapbox")
         .then(response => response.data)
```

Comment:
Choose yourself: since we're using TS, we should use typing as much as possible, and you can also type your axios stuff. The needed import is: `import axios, {AxiosResponse} from "axios";`

Every return from an axios request is of type `AxiosResponse` and in the body either the expected object from the body or without body just `void`. Helps you later on after unwrapping the data from the promise, everything is already typed and TS can do  type-checks.

Example:
```suggestion
        .then(response : AxiosResponse<Model of your access token>=> response.data)
```